Makefile

```diff
@@ -7,13 +7,15 @@ run:
 		redis \
 		mysql \
 		meilisearch \
+		datadog \
```

Comment:
Do we need to run the agent if we have it disabled?

Reply:
no, but its a PITA to conditionally run. my honest suggestion would be that the DD agent should be up to the person running the server but i imagine you don't want this